.travis.yml

```diff
@@ -1,10 +1,35 @@
 sudo: false
 dist: trusty
 language: node_js
+
 node_js:
```

Comment:
Why removing these? They should be running for all supported Node.js versions, right?

Reply:
Oh, you right. My bad.